geometry_msgs/Point sourcePosition
geometry_msgs/Point targetPosition
---
float64 distance